test/functional/apps/discover/_data_grid_doc_table.ts

```diff
@@ -38,10 +36,10 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await kibanaServer.uiSettings.replace({});
     });
 
-    it('should show the first 50 rows by default', async function () {
+    it('should show the first 12 rows by default', async function () {
       // with the default range the number of hits is ~14000
       const rows = await dataGrid.getDocTableRows();
-      expect(rows.length).to.be(defaultRowsLimit);
```

Comment:
@chandlerprall well, due to virtualization just 12 rows are displayed initially, curious if this is reliable or if we should test differently here

Reply:
Will be reliable based on browser size and/or DOM layout on the page. So it's stable, but sensitive to configuration changes.